### PR TITLE
Add Get TrackingPrograms ref data

### PR DESCRIPTION
**Pull request by scott062**

### CODE REVIEW

**Review by mjoyce91 — APPROVED** [review]

**Comment by mjoyce91** [comment]
@scott062 check that your Bureau Results page loads before merging

**Comment by scott062** [comment]
Recheck needed - added new GET route @elizabeth-jimenez

File: src/services/employees.js [hunk 1]
```diff
@@ -589,15 +589,15 @@ const get_assignments = async query => {
...
-      const perdet_seq_num = query['request_params.perdet_seq_num']
+      const perdet_seq_num = query['perdet_seq_num']
       if (perdet_seq_num) {
         qb.join('employees_classifications', 'employees_classifications.te_id', 'classifications.te_id')
         qb.where('employees_classifications.perdet_seq_num', perdet_seq_num)
       }
     }).fetchPage({
       require: false,
-      pageSize: query["request_params.page_size"] || 1000,
-      page: query["request_params.page_index"] || 1,
```
Comment (elizabeth-jimenez) [inline src/services/employees.js:600]: Should we remove `request_params ` from `add_classification` and `remove_classification` too then?

**Review by elizabeth-jimenez — APPROVED** [review]
👍

**Pull request merged**